tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor/Constants/RulesConstants.cs

```diff
@@ -48,6 +48,7 @@ public class RulesConstants
         public const string IdentifyStalePullRequests = "IdentifyStalePullRequests";
         public const string CloseAddressedIssues = "CloseAddressedIssues";
         public const string LockClosedIssues = "LockClosedIssues";
+        public const string Enforce2YearMaxLifeOfIssues = "Enforce2YearMaxLifeOfIssues";
```

Comment:
Should we maybe generalize the name a bit, in case we adjust the "2 year" duration at a later point?   It would suck to have to go update all the repo config files or have the name and rule mismatch.

Reply:
@jsquire changed the rule to EnforceMaxLifeOfIssues which is more generic.